doc/nrf/protocols/multiprotocol/index.rst

```diff
@@ -46,3 +46,60 @@ Given the nature of the dynamic multiprotocol solution, take into account the fo
   The application logic must take this into account by providing a callback for all crucial transmissions and handle the ``MAC_TRANSACTION_OVERFLOW (== 241)`` status code.
 * The most time-critical part of the Zigbee stack operation is the Base Device Behavior (BDB) commissioning process.
   Postpone time-consuming operations until the ``ZB_BDB_SIGNAL_DEVICE_FIRST_START`` or ``ZB_BDB_SIGNAL_DEVICE_REBOOT`` signal is received.
+
+.. _BT_Thread_coex:
+
+Bluetooth LE and Thread coexistence
+###################################
+
+Radio Time Management Between Bluetooth LE and Thread
+*****************************************************
+
+The radio is dynamically switched between Bluetooth LE and Thread tasks, allowing active communication in both protocols without the need for two separate radio chips.
+However, Thread has a lower priority compared to Bluetooth LE, In scenarios where Bluetooth LE consumes a significant amount of radio time,
+such as during scanning or maintaining a big number of Bluetooth LE connections, there may be insufficient time left for Thread.
+Consequently, Thread may attempt to communicate during the same time slots allocated to Bluetooth LE, but since it has a lower priority, it might not get access to the radio.
+In such situations, Thread communication can become inefficient or even be lost entirely.
+To maintain effective Thread communication, it is essential to reduce the radio time for Bluetooth LE. This can be achieved by adjusting the connection and scan parameters.
+
+Bluetooth LE Scan Parameters
+****************************
+
+The key parameters for scanning are
+
+- *scan interval* (the time between consecutive scans).
+- *scan window* (the amount of time within the scan interval that is dedicated to actual scanning).
+
+By reducing the scan window for a given scan interval, the actual radio time used by Bluetooth LE decreases, thereby freeing up more time for Thread.
+Analogously increasing the scan interval for a given scan window, there will be same effect.
```

Comment:
I would add somewhere the sentence that it's actually about the % of time that is occupied by Bluetooth LE. However, the user has to be careful, as using 20 ms window and 60 ms interval (33% of used time) will not be the same as using 100 ms window and 300 ms interval (also 33%). This is because period of time constantly occupied by BLE for active scanning should not be too big, as probability of Thread transmissions hitting this part will be bigger.